lib/game.rb

```diff
@@ -0,0 +1,36 @@
+class Game
+
+  def initialize(player_1_choice, player_2_choice)
+    @player_1_choice = player_1_choice.downcase.capitalize
+    @player_2_choice = player_2_choice.downcase.capitalize
+  end
+
```

Comment:
The game logic is duplicated here, in the two play-related methods, which means a change to the rules would always have to involve updating two methods and that might lead to some mistakes.

Can you figure out a way to reduce duplication?